Documentation/project-docs/writing-tests.md

```diff
@@ -0,0 +1,93 @@
+This document provides an overview of paradigm we are using in our tests.
+
+# RemoteExecutor
```

Comment:
> Not true. It's very useful for:

I made the wrong assumption given that the exe is part of System.Diagnostics and I've only seen it fail during EventSource tests. Your explanation could be useful as a part of the proposed documentation.